REPOSITORY-GUIDELINES.md

```diff
@@ -0,0 +1,131 @@
+
+### ```DEPRECATED```
+
+Deprecated repositories are those that are no longer maintained and are only kept for historical purposes. They are not recommended for use by adoptors. They are typically [archived](#archival) and read-only.
+
+## Lifecycle
+
+### Addition
+
+Repositories of type ```Ecosystem```, ```Infra```, or ```Special``` can be added into the GitHub organization at the discretion of the Cilium committers (in other words, by lazy consensus).
+
+Any new repository of any type must be added into the appropriate list in [REPOSITORIES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md) and be assigned a status (if applicable).
+
+#### Adding ```Core``` repositories
+
+Given their importance to the project, repositories of the ```Core``` type must be added through a more formal process:
+  - The new project is proposed by a committer via a GitHub issue discussion in one of the existing projects in the organization. Part of the discussion should include whether the proposed project [meets the requirements](#stable) to be assigned a ```STABLE``` status.
+  - The discussion is open to all, and people in the community are encouraged to participate. While voting is exclusive to committers, this discussion is useful for those with voting power to understand the perception of the wider Cilium community.
+  - Once sufficient discussion has taken place (for example, 3-5 business days), the committers decide if it should be added via a simple majority vote.
+  - If the proposed project is accepted and added to the Cilium GitHub organization, it is then added to the list in [REPOSITORIES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md#core-scope) and is assigned the ```STABLE``` status.
+
+### Change of scope or status
+
+As a repository increases or decreases in activity, it may become appropriate to pursue a change in status or scope. For most cases, this can be done via lazy consensus by a Cilium committer. 
+
+One exception is when pursuing a ```Core``` scope, upon which the process described in [Adding ```Core``` repositories](#adding-core-repositories) is followed.
+
+#### Archival
+
+Repositories that have shown little activity in over a year may be a good candidate for archival. Archived repositories are given the ```DEPRECATED``` status, are read-only, and are no longer maintained.
+
+If they hold historical or other useful purposes, they will remain as an archived repository within the Cilium GitHub organization.
+
+If they have little relevance or purpose to add to the project, they can be moved to the [Cilium Projects Retirement Home](). This ensures that there continues to be a recorded history of the project while also ensuring only relevant repositories are kept within the main Cilium organization.
```

Comment:
Lines 125 & 127 can be taken out if we don't want to pursue the retirement home option